### DISCUSSION TITLE
Burn Bootloader with AVRDUDE doesn't set fuses

### DISCUSSION THREAD

**Discussion by pcfreak1201** [ArminJo/DigistumpArduino#17] [General]
Moin Armin!

The new features (1.7.3) work fine, if you only want to change bootloader.
But you can't generate working USB devices from scratch / new chips, because the fuses won't be set
`.arduino15/packages/arduino/tools/avrdude/6.3.0-arduino18/bin/avrdude -C/home/stefan/.arduino15/packages/digistump/hardware/avr/1.7.3/avrdude.conf -v -pattiny85 -cusbasp -Uflash:w:/home/stefan/.arduino15/packages/digistump/hardware/avr/1.7.3/bootloaders/upgrade-t85_entry_on_powerOn_activePullup_fastExit.hex:i `
ATTinyCore:
`.arduino15/packages/arduino/tools/avrdude/6.3.0-arduino18/bin/avrdude -C/home/stefan/.arduino15/packages/ATTinyCore/hardware/avr/1.5.2/avrdude.conf -v -pattiny85 -cusbasp -e -Uefuse:w:0xFE:m -Uhfuse:w:0b11011{bootloader.bod_bits}:m -Ulfuse:w:0xF1:m -Uflash:w:/home/stefan/.arduino15/packages/ATTinyCore/hardware/avr/1.5.2/bootloaders/micronucleus/t85_entry_on_power_on_no_pullup_fast_exit_on_no_USB.hex:i`
Is this missing, or a security feature?

**Comment by ArminJo** [comment]
Moin pcfreak1201
The feature to set the fuses is intentional missing. I fear, that it makes more problems, than it solves. And everybody buying a chip with "wrong" fuses should know how to change them.
But thanks for mentioning, I added a chapter Fuse settings in the Readme.

**Reply by pcfreak1201** [reply]
Ah, dachte ich mi schon ...
